manifest.json

```diff
@@ -1,15 +1,15 @@
 {
   "vendor": "vtex",
-  "name": "base-store-component",
+  "name": "payment",
```

Comment:
I think it's better to use `checkout-payment`, so we don't get confused by the VTEX Payment project.